### ISSUE TITLE
Why set only 5 epochs in training MSR-VTT.

### ISSUE THREAD

**Issue by BlueCat7** [ArrowLuo/CLIP4Clip#10]
Hi,
   When I training more epochs in mar-cry, it seems much better. And why do you 5 epochs.Thanks.

**Comment by ArrowLuo** [comment]
Hi @BlueCat7, the numbers 3 and 5 are common epochs at the finetune phase. We used 5 in this paper and did not try different epochs. We roughly set the common hyper-parameters across the datasets and believe the performance will be better via careful adjustment of hyper-parameters on each dataset. 

Would you paste your epoch and results? Thanks.

**Comment by BlueCat7** [comment]
I only have about 700 testing videos for some urls have been broken. When I training 5 epoch and get 41.1 R@1 in text2video of MSR-VTT dataset. 64.8 R@1 at 97 epoch, and more epochs, the accuracy do not get better.

**Comment by ArrowLuo** [comment]
Thank you for your sharing. The improvement is higher than expected. We are testing more epochs on our dataset and any information will be shared here. The time may be longer if we run 100 more epochs.

**Comment by ArrowLuo** [comment]
Hi @BlueCat7, can you share your log.txt? We have run 20 epochs on MSRVTT and 100 epochs on other datasets, and the performance does not become much better for excepted. I wonder there are some differences between your settings and ours besides the number of datasets.

**Comment by ArrowLuo** [comment]
There is no significant gain after training with more epochs in our setting. Feel free to reopen this issue if any other information.

**Comment by starmemda** [comment]
> I only have about 700 testing videos for some urls have been broken. When I training 5 epoch and get 41.1 R@1 in text2video of MSR-VTT dataset. 64.8 R@1 at 97 epoch, and more epochs, the accuracy do not get better.

Could you share your settings? Our experiments didn't show such performance.

**Comment by ArrowLuo** [comment]
Hi @starmemda, I think you want to ask for help from @BlueCat7. Thanks.
